api/apps/api/src/modules/scenarios/scenarios.controller.ts

```diff
@@ -213,9 +234,20 @@ export class ScenariosController {
   @Get(':id/planning-units/tiles/:z/:x/:y.mvt')
   async proxyProtectedAreaTile(
     @Req() request: Request,
+    @Req() req: RequestWithAuthenticatedUser,
```

Comment:
Do we really need the request twice here? The arg you added extends Request, so is there any reason why it can't simply replace the existing Request?

Reply:
my fault (mindlessly accepting IDE suggestions) - `RequestWithAuthenticatedUser` is a remnant of past projects, and I brought it over importing the wrong `Request`, so fixing `RequestWithAuthenticatedUser` to extend `Request` from Express is the correct way to handle this.